src/api/ZoweExplorerApi.ts

```diff
@@ -43,6 +43,15 @@ export namespace ZoweExplorerApi {
          * @returns {Session} a Zowe CLI Session
          */
         getSession(profile?: IProfileLoaded): Session;
+
+        /**
+         * Create a session for the specific profile type.
+         *
+         * @param {IProfileLoaded} profile
+         *      will use the profile the API was retrieved with by default
+         * @returns {IZosmfInfoResponse} z/OSMF Check Status response
+         */
+        getStatus(profile: IProfileLoaded, profileType?): Promise<string>;
```

Comment:
@jellypuno the new getStatus should have a ? after it like getStatus?(profile: IProfileLoaded, profileType?): Promise<string>; until v2.0 so if the extender doesn't have it yet it will be optional and the caller cannot just call it, but they need to test if it exists; so on the Zowe Explorer side they need something like if (ZoweExplorerApiRegister.getUssApi(node.getProfile()).getStatus) { ZoweExplorerApiRegister.getUssApi(node.getProfile()).getStatus()}

Reply:
ok. I will change it as you said.